subsys/net/lib/aws_jobs/Kconfig

```diff
@@ -5,7 +5,7 @@
 
 menuconfig AWS_JOBS
 	bool "AWS Jobs library"
-	depends on ENTROPY_DEVICE_RANDOM_GENERATOR
+	depends on (ENTROPY_DEVICE_RANDOM_GENERATOR || BOARD_NATIVE_POSIX)
```

Comment:
I guess the whole line could be removed now that https://github.com/nrfconnect/sdk-nrf/pull/2999 is merged.